docs/source/cli-reference/synopsis.rst

```diff
@@ -7,25 +7,10 @@ ScanCode detects licenses, copyrights, package manifests and direct dependencies
 in source code and binary files, by scanning the files. This page introduces you to the ScanCode
 Toolkit Command Line Interface in the following sections:
 
-- Installation
-- Quickstart
-- Type of Options
-- Output Formats
-- Other Important Documentation
-
-.. _syn_install:
```

Comment:
I think this part should remain, as it's helpful and just references to installation which is an pre-requisite to using the CLI, and this is just references anyway.